error/call2.luau:2: function (main)
attempt to call a number value